### ISSUE TITLE
Volcanic Ash report needs to be updated

### ISSUE THREAD

**Issue by Charlotte-Morgan** [inasafe/inasafe-realtime#136]
## problem ##
There is nothing on the pdf InaSAFE Volcanic Ash report to say what the duration of the ash forecast is.

## proposed solution ##
header information on the report needs to be updated to include if the forecast is a 1 day or 3 day forecast. 
(also need to fix Eruption column to a new line)
![image](https://user-images.githubusercontent.com/5743135/33248367-b8584ace-d378-11e7-8d0b-c16a1bd5bdd5.png)

This information needs to be added to the upload dialog
![image](https://user-images.githubusercontent.com/5743135/33248337-85a03416-d378-11e7-9cc9-c89d6d45cd56.png)

## cc ##
@RikkiWeber @gubuntu @estukriswati

**Comment by Charlotte-Morgan** [comment]
Hi @lucernae  - I have received further requests to update the Volcanic ash report to remove any ambiguity for people reading the report. This requires more text to be included on the report before we can share it to others.
Making these changes is a high priority. Could you please provide advice on the template and the best way for changes to the template and text messages.

cc
@timlinux @ivanbusthomi @myarjunar

**Comment by lucernae** [comment]
maybe mas @ivanbusthomi can help on this one.
what I can suggest is to create a new placeholder column, probably below the location.
so, you need to add [eruption column] and [forecast duration].

<img width="591" alt="screen shot 2017-11-29 at 15 13 45" src="https://user-images.githubusercontent.com/831865/33364673-ffa8abe4-d517-11e7-9e76-01a0e09681cf.png">

Then you can share the template to me.

For the forecast duration field, I need some more info:
- type of field: was it a combo box with options, or people can put arbitrary integer number there?
- the default value of the field.

**Comment by Charlotte-Morgan** [comment]
Hi @lucernae 
I am happy to work with @ivanbusthomi to update the template for fields and text string changes.
For the new information on the upload form, I suggest we need a combo box with options for 1 day or 3 day. These are the 2 standard options for the forecast duration.
@RikkiWeber - pls correct me if i am wrong

**Comment by Charlotte-Morgan** [comment]
Hi @ivanbusthomi - this is a high priority and if possible I would like to have changes to the template implemented on Tuesday 5th December

**Comment by lucernae** [comment]
mas @ivanbusthomi I will give you the btsync key in PM

**Comment by ivanbusthomi** [comment]
Hi mas @lucernae - this is the screenshot of the updated template. it is still need improvement and some tidying up. I already add a placeholder for the column height element. But since there is no token for that, it still shows the placeholder, maybe you can add it.
![image](https://user-images.githubusercontent.com/7351977/34240749-499cca28-e643-11e7-9634-3f37132dceb2.png)

I also reserve a bit whitespace under the population table to accommodate the size of the table in indonesian report which is a little bit bigger than the english one.

**Comment by lucernae** [comment]
mas @ivanbusthomi you can look for available token in:

https://github.com/inasafe/inasafe-realtime/blob/realtime-backport-cherry-pick/realtime/ash/report_text.py

Can you edit the token name in template to each relevant variable available in that file.
I also provided more atomic token in case you need it, like the eruption-height-asl or eruption-height.

**Comment by ivanbusthomi** [comment]
Thanks @lucernae for the update on the report creation. this is the pdf report of test event for Lokon volcano at 1st of Januari 2018 01:00 Central Indonesia Time.
![image](https://user-images.githubusercontent.com/7351977/35557751-9a9e6512-05d8-11e8-8421-84a1cc2986a2.png)

I'd like to highlight the changes implemented here which are:
* fixing the eruption height calculation to includes the vent height
* adding additional information on the event time to also include the time zone name and difference with UTC
* fixing a bug where the timezone information is not showing correct event time information as noticed by Ibu @estukriswati .

I will create a new ticket for me to fix the overlapping text at the volcano and time information.
CC @Charlotte-Morgan

**Comment by Charlotte-Morgan** [comment]
Hi @ivanbusthomi  and @lucernae  - I am still not seeing the duration in the header of the report

cc @gubuntu

**Comment by lucernae** [comment]
@Charlotte-Morgan Ah, sorry. I think I miss the forecast duration. I will try to complete it now.

**Comment by lucernae** [comment]
@Charlotte-Morgan @ivanbusthomi 
Is this mockup looks good?
<img width="739" alt="screen shot 2018-02-01 at 17 16 11" src="https://user-images.githubusercontent.com/831865/35673281-aa7a6fb6-0773-11e8-9b25-dde10ffa7e42.png">

**Comment by ivanbusthomi** [comment]
Hmmm, I am thinking a increase-decrease value like we have in eruption height input. But the one that you create is OK too I guess
What do you think @Charlotte-Morgan ?

It means we also need to add this as an extra keyword also mas @ismailsunni ?

**Comment by lucernae** [comment]
Well, @Charlotte-Morgan mentioned that it probably in the form of combo box.

> For the new information on the upload form, I suggest we need a combo box with options for 1 day or 3 day. These are the 2 standard options for the forecast duration.

But, it is easy to change it into increase-decrease value. Just tell me what needs to be done.

**Comment by Charlotte-Morgan** [comment]
Hi @lucernae  - the duration combo box should have only 2 options. 1 day and 3 days.

This is based on the weather forecast that the ash modellers use to create the model.
@RikkiWeber

**Comment by ismailsunni** [comment]
> It means we also need to add this as an extra keyword also mas @ismailsunni ?

Just let me know what I need to add after the discussion finished :D

**Comment by lucernae** [comment]
mas @ivanbusthomi you can try the new changes for forecast duration.
the new variable is here:

https://github.com/inasafe/inasafe-realtime/blob/bddc1f6c65df096002bca53e7c4c2624e7961cfe/realtime/ash/report_text.py#L28-L41

**Comment by ivanbusthomi** [comment]
I am updating the template to also include the `event-forecast-duration`

**Comment by ivanbusthomi** [comment]
Hi @Charlotte-Morgan 
This is my attempt at tweaking the template to include the model duration and expiry time. I've taken your suggestion in https://github.com/inasafe/inasafe-realtime/issues/142#issuecomment-348393199 in terms of the header element, but I've changed the arrangement a little bit so that all of the element can fit.
* header
![image](https://user-images.githubusercontent.com/7351977/35841076-a59e541e-0b2c-11e8-86ca-44461eef1579.png)

* nearby table
 affected
![image](https://user-images.githubusercontent.com/7351977/35841442-984b3d52-0b2e-11e8-8bbe-08b9f4e35866.png)
no affected
![image](https://user-images.githubusercontent.com/7351977/35841408-69194ab0-0b2e-11e8-9081-0fd5191c2c12.png)

* disclaimer
![image](https://user-images.githubusercontent.com/7351977/35841418-762a681a-0b2e-11e8-83dd-795a7d538371.png)


@gubuntu @lucernae @ismailsunni @estukriswati

**Comment by Charlotte-Morgan** [comment]
hi @ivanbusthomi  - as dicussed. I am happy with the arrangements you have made to the header to fit the information in & the way you have handled the disclaimer text.

I have discussed the changes with @RikkiWeber and she is also happy with this template
THANK YOU :)

**Comment by Charlotte-Morgan** [comment]
I think we can close this
